### ISSUE TITLE
[QUESTION]   Any samples or suggestions for generating aliases.jsonl?

### ISSUE THREAD

**Issue by moutaigua8183** [microsoft/spacy-ann-linker#2]
I am a first-time user of entity linking. I kind of know how to generate entities.jsonl, but am wondering how to generate a new aliases.jsonl from my data.

**Comment by jack-rory-staunton** [comment]
Yes I too am curious about this. It seems to me very few people are using spacy knowledgebase and entity linking functionalities because there is no good spacy documentation on how to set anything up. Also, I was hoping for more of a corpus-intrinsic entity resolution functionality that doesn't depend on already having entities deduplicated and canonicalized.
